cleaning/big_clean.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Robert Beane
+
+# Parameters
+comp=$1 # takes inputed file
```

Comment:
What does `comp` mean as a name here?

Reply:
I think I was thinking compressed, I don't like it either anymore. Will change to something that makes more sense and add a comment describing.